### PR TITLE
TASK: Use Open Sans as the default UI font

### PR DESCRIPTION
**Pull request by dfeyer**

### CODE REVIEW

**Comment by dfeyer** [comment]
Require https://github.com/neos/brand/pull/8 to be merged and a new version of the package released

File: packages/build-essentials/src/webpack.config.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2|eot|ttf|svg)$/,
+                loader: 'url-loader?limit=10000&name=./Styles/Font-[hash].[ext]&publicPath=./../'
```
Comment (dfeyer) [inline packages/build-essentials/src/webpack.config.js]: A bit ugly, but I'm not a big friend of Webpack, so maybe someone found a better solution

Comment (dfeyer) [reply packages/build-essentials/src/webpack.config.js]: I love the value of limit, it does not make sens to embed the font in the CSS; HTTP2 make a better job with concurent request, and we don't update the font a lots ;)

File: packages/react-ui-components/.storybook/webpack.config.js [hunk 2]
```diff
@@ -13,8 +13,8 @@ module.exports = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2)$/,
-                loader: 'url?limit=100000'
+                test: /\.(png|woff|woff2|eot|ttf|svg)$/,
+                loader: 'url?limit=10000'
```
Comment (dfeyer) [inline packages/react-ui-components/.storybook/webpack.config.js]: Not sure if this one need to be adapted, not tested

File: packages/neos-ui/src/styleHostOnly.css [hunk 3]
```diff
@@ -1,25 +1,81 @@
...
+    font-weight: 300;
+    font-style: normal;
 }
 
 @font-face {
-  font-family: 'Work Sans';
-  src: url('../../../Resources/Public/Fonts/WorkSans-Bold.woff') format('woff');
-  font-weight: bold;
-  font-style: normal;
+    font-family: 'Open Sans';
```
Comment (dimaip) [inline packages/neos-ui/src/styleHostOnly.css]: Do we need all of those font weights? That would be like tons of data...

Comment (dfeyer) [reply packages/neos-ui/src/styleHostOnly.css]: @dimaip Based on the discussion on the UX Guild, I think we need more weights in the future. Currently we use only 2 weight (regular and bold), so only used weights are loaded.

Before this change the font is base64 encoded in the CSS, so the CSS was around 500Ko, now we have a really slim CSS, and even if all weights are loaded we save bandwidth. Fonts can be cached (http, or localstorage in the future, if we need to improve this)

File: packages/build-essentials/src/webpack.config.js (hunk 1 above)
Comment (skurfuerst) [reply packages/build-essentials/src/webpack.config.js]: ack :)

File: packages/neos-ui/src/styleHostOnly.css (hunk 3 above)
Comment (dimaip) [reply packages/neos-ui/src/styleHostOnly.css]: @dfeyer hey, but let's add font weights only when we need them, when the design changes? We ship something when it's not used?
Currently we don't base64 fonts afaik, @mstruebing fixed it a while ago.

Comment (dimaip) [reply packages/neos-ui/src/styleHostOnly.css]: @dfeyer do you agree?

**Comment by dimaip** [comment]
@dfeyer hey, do you still plan to work on this one?

**Comment by skurfuerst** [comment]
We should implement this really soon :) @dimaip would you have time to do this? :)

**Comment by dimaip** [comment]
Yes!

**Comment by dimaip** [comment]
Don't seem to have push access here, closing in favor of #1089

**Pull request closed without merging**